backend/backend/core/tools.py

```diff
@@ -1,4 +1,5 @@
 import os
+import re
```

Comment:
The import statement for 're' is not used in this code patch. Consider removing it.